eox_nelp/apps.py

```diff
@@ -77,6 +77,14 @@ class EoxNelpConfig(AppConfig):
                         'receiver_func_name': 'update_async_tracker_context',
                         'signal_path': 'celery.signals.task_prerun',
                     },
+                    {
+                        'receiver_func_name': 'emit_subsection_attempt_event',
+                        'signal_path': 'lms.djangoapps.grades.signals.signals.PROBLEM_WEIGHTED_SCORE_CHANGED',
```

Comment:
This would trigger with verticals or units attempts? or only units(verticals) related graded  subseection(sequential) ?

Reply:
https://github.com/openedx/edx-platform/blob/4b7ef2697e84bae269983dfce63f1af60bb247df/lms/djangoapps/grades/signals/signals.py#L34